### PR TITLE
Ship prometheus check in Agent 6

### PR DESCRIPTION
**Pull request by ISauve**
### What does this PR do?

Adds `prometheus_check.py` to the agent dist folder along with the protobuf bits it requires.

### Motivation

Prometheus checks such as kube_dns and kube-state-metrics (and more to come) need the prometheus_check.py file which serves as the base class for them.

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: cmd/agent/dist/checks/prometheus/functions.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# (C) Datadog, Inc. 2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+from google.protobuf.internal.decoder import _DecodeVarint32  # pylint: disable=E0611,E0401
```
Comment (truthbk) [inline cmd/agent/dist/checks/prometheus/functions.py]: we'll need changes to omnibus to make sure this ships as well with the core.

File: cmd/agent/dist/checks/prometheus/metrics_pb2.py [hunk 2]
```diff
@@ -0,0 +1,588 @@
+# flake8: noqa
+# pylint: skip-file
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: metrics.proto
+
+import sys
+_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
+from google.protobuf.internal import enum_type_wrapper
```
Comment (truthbk) [inline cmd/agent/dist/checks/prometheus/metrics_pb2.py]: ditto regarding omnibus

File: omnibus/config/software/datadog-agent-integrations.rb [hunk 3]
```diff
@@ -19,7 +19,6 @@
   'agent_metrics',
   'docker_daemon',
   'kubernetes',
-  'kubernetes_state',
```
Comment (truthbk) [inline omnibus/config/software/datadog-agent-integrations.rb]: I can't remember, has this been deprecated?

**Review by truthbk — COMMENTED** [review]
I think we're missing the omnibus part. Other than that, this looks good. Once you confirm re:omnibus I'll give it another pass.

File: omnibus/config/software/datadog-agent-integrations.rb (hunk 3 above)
Comment (masci) [reply omnibus/config/software/datadog-agent-integrations.rb]: it seems alive and kickin' https://github.com/DataDog/integrations-core/tree/master/kubernetes_state but I'm not sure whether we're going to kill it or what

Comment (xvello) [reply omnibus/config/software/datadog-agent-integrations.rb]: indeed, 6.0 needs to ship with it

as the ksm payloads get bigger and bigger (12k metrics in some clusters), we might investigate reworking it, but that's 2018 material

**Review by masci — APPROVED** [review]
🚀

**Pull request merged**